apps/ffdweb-site/src/app/_components/Footer.tsx

```diff
@@ -0,0 +1,80 @@
+import { SmartTextLink } from '@filecoin-foundation/ui/TextLink/SmartTextLink'
+import clsx from 'clsx'
+
+import { footerLegalItems, footerNavigationItems } from '@/constants/navigation'
+import { BASE_DOMAIN } from '@/constants/siteMetadata'
+
+import { Logo } from '@/components/Logo'
+import { Social } from '@/components/Social'
+
+const FOOTER_GAP = 'gap-16'
+const FOOTER_SECTION_GAP = 'gap-10'
+
+export function Footer() {
+  return (
+    <footer className={clsx('grid grid-cols-1', FOOTER_GAP)}>
+      <div
+        className={clsx(
+          'grid grid-cols-1 lg:auto-cols-max lg:grid-flow-col',
+          FOOTER_GAP,
+        )}
+      >
+        <Logo />
+        <nav
+          aria-label="Footer navigation"
+          className={clsx(
+            'grid grid-cols-1 sm:auto-cols-fr sm:grid-flow-col',
+            FOOTER_GAP,
+          )}
+        >
+          {Object.entries(footerNavigationItems).map(([title, items]) => (
+            <div
+              key={title}
+              className={clsx('grid content-start', FOOTER_SECTION_GAP)}
+            >
+              <h3 className="text-base font-bold capitalize">{title}</h3>
+              <ul className="grid gap-3">
+                {items.map(({ href, label }) => (
+                  <li key={href}>
+                    <SmartTextLink href={href} baseDomain={BASE_DOMAIN}>
```

Comment:
When we know that the links are internal we can use `InternalTextLink` directly (if the styles align to what we need) or next.js `Link` (if we need custom styles)

Reply:
Also, according to the design, all these link are white not green. 🙏🏼